package.json

```
@@ -85,6 +95,7 @@
     "file-loader": "4.2.0",
     "glob": "7.1.4",
     "html-webpack-plugin": "3.2.0",
+    "instantsearch-e2e-tests": "algolia/instantsearch-e2e-tests#v1.2.1",
```

Last release is v1.2.2.

```suggestion
    "instantsearch-e2e-tests": "algolia/instantsearch-e2e-tests#v1.2.2",
```